WcaOnRails/db/seeds/round_types.seeds.rb

```diff
@@ -6,10 +6,10 @@
 ('2', 50, 'Second round', 'Second', 0),
 ('3', 79, 'Semi Final', 'Semi Final', 0),
 ('b', 39, 'B Final', 'B Final', 0),
-('c', 90, 'Combined Final', 'Combined Final', 1),
-('d', 20, 'Combined First round', 'Combined First', 0),
-('e', 59, 'Combined Second round', 'Combined Second', 0),
+('c', 90, 'Cutoff Final', 'Cutoff Final', 1),
```

Comment:
I see cutoff being used here both in name and cell name, which is not the case in `en.yml`. Whatever we choose we should be consistent.